@stellar/design-system-website/src/componentPreview/buttonPreview.tsx

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { ComponentPreview } from "@site/src/components/PreviewBlock";
 import CheckIconSvg from "@site/static/img/check-icon.svg";
 
-export const ButtonPreview: ComponentPreview = {
+export const buttonPreview: ComponentPreview = {
```

Comment:
I renamed preview files and names to start with a lowercase letter because they are a config file and not a component.